src/modules/ChannelSettings/components/ChannelProfile/index.tsx

```diff
@@ -28,10 +28,10 @@ const ChannelProfile: React.FC = () => {
   const channel = channelSettingStore?.channel;
 
   const channelName = useMemo(() => {
-    if (channel?.name && channel.name !== 'Group Channel') {
+    if (channel?.name && channel.name !== 'Group Channel' && channel.name !== 'AI Chatbot Widget Channel') {
```

Comment:
Let's manage this `AI Chatbot Widget Channel` string separately in somewhere else so we don't need to compare the string directly every time.

Reply:
Also, shall we create a separate util function for getting this condition ? 
```
if (channel?.name && channel.name !== 'Group Channel' && channel.name !== 'AI Chatbot Widget Channel') {
```

Reply:
`isDefaultChannelName()` is added in `src/utils/index.ts`. Also, `DEFAULT_GROUP_CHANNEL_NAME` and `DEFAULT_AI_CHATBOT_CHANNEL_NAME` are introduced too. But those consts are only used in `isDefaultChannelName()` for now.